### ISSUE TITLE
Anki Redesign on Anki 2.1.22

### ISSUE THREAD

**Issue by Willz7** [Shirajuki/anki-redesign#17]
Once again, thank you for keeping this Add-on up to date and making Anki more contemporary and attractive.

When I tried to modify several of the colour schemes in Anki 2.1.22, I received this error message below.

```
Debug info:
Anki 2.1.22 (0ecc189a) Python 3.8.1 Qt 5.12.6 PyQt 5.12.3
Platform: Windows 10
Flags: frz=True ao=True sv=2
Add-ons, last update check: 2022-04-13 07:55:40

Caught exception:
Traceback (most recent call last):
  File "C:\Users\William\AppData\Roaming\Anki2\addons21\308574457\__init__.py", line 507, in <lambda>
    button.clicked.connect(lambda _: self.save())
  File "C:\Users\William\AppData\Roaming\Anki2\addons21\308574457\__init__.py", line 541, in save
    showInfo(tr.preferences_changes_will_take_effect_when_you())
AttributeError: 'function' object has no attribute 'preferences_changes_will_take_effect_when_you'
```

**Comment by Shirajuki** [comment]
Oh no!
I guess I was too quick on my manual testing last night.
I'll be pushing a new fix for this shortly! I'll update you here when it's fixed!

Again, I want to express that I am very happy to see that Anki-redesign has been of use to someone other than me as well :D
Thank you for being a user of Anki-redesign! 🥳

**Comment by Shirajuki** [comment]
Hello again! I believe that this issue has been fixed now with the new small update. I didn't know that the function `tr.preferences_changes_will_take_effect_when_you()` was introduced in the newer versions, and have now changed it to use custom text now, hopefully this fix of update (v0.1.1) will work for you!

There is also one visual bug on the older versions where the TabWidget background doesn't change background color with the palette / colors that is set. I'll look into it in a further update :D. Feel free to reopen this issue/create a new one if there are problems that still exists.
